### DISCUSSION TITLE
Give me Legguards of Endless Rage

### DISCUSSION THREAD

**Discussion by vvranceanu** [chromiecraft/chromiecraft#7272] [Suggestions]
I have been raiding Hyjal for 3 months(only clearing for 1), and haven't seen https://www.wowhead.com/tbc/item=30903/legguards-of-endless-rage even drop. I think 3 months is a fair amount of time to try (success doesn't matter, participation is key) to get an item, so at this point I kinda expect my bis pants.

I've seen discussions about Mythic Points, but let's not beat around the bush, why jump through all those hoops when the result is exactly the same? This is a game i play to feel good and i need the pants to feel better. I only think this is fair and my opinion is super important(mom always listens)

Thank you for you time!

**Comment by BeefSteaker** [comment]
Please keep suggestions constructive. This is an obvious troll based on your comments in the theorycrafting channel

**Comment by shaszun** [comment]
Give this man his pants!

**Comment by Shardaz** [comment]
Just implement a vendor that sells all drops from the respective raid after you kill the last boss. This would actually help casual players like him to get all the loot after a single clear. Hardcore MinMaxers can still save some gold by clearing the raid multiple times. Clear WinWin

**Reply by vvranceanu** [reply]
yoooooooooooooooo this is clearly a well thought constructive(not based on a whim, half baked) suggestion and you seem like a capable individual yourself(names starting with S are powerful names!). you should make a discussion post imo. opinions are important and every single one needs to be heard and debated!
